app/controllers/admin/motifs_controller.rb

```diff
@@ -100,4 +101,8 @@ def motif_params
               :follow_up,
               :sectorisation_level)
   end
+
+  def set_available_services
+    @available_services = current_organisation.territory.organisations.flat_map(&:agents).flat_map(&:service).uniq
```

Comment:
Je crois qu’on peut l’écrire comme ça pour éviter de faire N allers-retours avec la DB:
```suggestion
    @available_services = Service.where(agents: Agent.joins(:organisations).merge(current_organisation.territory.organisations))
```

(J’ai un projet de patch sur activerecord pour pouvoir écrire directement `current_organisation.territory.organisations.through_query.agents.service` et faire la bonne requête, mais c’est pas trivial.)